### ISSUE TITLE
Upload gradle job fails on Windows, possibly missing backslash escape

### ISSUE THREAD

**Issue by sam-vdp** [getsentry/react-native-sentry#304]
OS:
- [x]  Windows
- [ ]  MacOS
- [ ]  Linux

_Platform:_
- [ ]  iOS
- [x]  Android

Output of `node -v && npm -v && npm ls --prod --depth=0`

```
v8.4.0
5.5.1
[...]
+-- react-native-sentry@0.30.2
```

Config:
Not relevant

---
I have following issue:

gradle task :app:bundleReleaseJsAndAssetsreleaseSentryUpload fails with error `'node_modulessentry-cli-binarybinsentry-cli' is not recognized as an internal or external command, operable program or batch file.`

Note the missing slashes in the command string. The android/sentry.properties contains the following line:
`cli.executable=node_modules\sentry-cli-binary\bin\sentry-cli`
If I replace the singles backslashes in this string with double backslashes, the task runs successfully. However, this will probably break when building on MacOS.

**Steps to reproduce:**
npm install react-native-sentry
react-native link (enter configuration)
cd android
gradlew assembleRelease

**Actual result:**
app:bundleReleaseJsAndAssetsreleaseSentryUpload fails

**Expected result:**
app:bundleReleaseJsAndAssetsreleaseSentryUpload  succeeds

**Comment by sam-vdp** [comment]
Some more detail:
sentry.gradle loads the .properties file like this:
```
            Properties sentryProps = new Properties();
            try {
                sentryProps.load(new FileInputStream(propertiesFile));
            } catch (FileNotFoundException e) {
                println "File not found: $propertiesFile"
            }
            def cliExecutable = sentryProps.get("cli.executable", "$reactRoot/node_modules/sentry-cli-binary/bin/sentry-cli");
```
As noted for the .properties file format
> The backwards slash is used to escape a character
(https://en.wikipedia.org/wiki/.properties)

The issue can fixed by simply removing the cli.executable property from the .properties file entirely, the default value then works.

**Comment by HazAT** [comment]
Hey, thank you for pointing that out.
I've released a new version of react-native-sentry `0.31.0` where this should be resolved.
